src/graph.js

```diff
@@ -60,6 +60,12 @@ export class Graph {
     return new Nodes(this, nids)
   }
 
+  /**
```

Comment:
Does the docstring add anything useful? I think docs like this are a love/hate thing. The comments can easily get out of sync with the actual types involved, too.